fake_switches/juniper/juniper_netconf_datastore.py

```diff
@@ -495,3 +500,9 @@ def get_or_create_interface(if_list, port):
 
     return existing
 
+
+def port_is_in_access_mode(port):
+    return port.mode is None or port.mode == "access"
```

Comment:
When it's not set:
https://github.com/mat128/fake-switches/blob/juniper_cannot_remove_last_trunk_member/fake_switches/juniper/juniper_netconf_datastore.py#L251